step-capstone/src/components/Trip.js

```diff
@@ -16,23 +16,35 @@ export default class Trip extends React.Component {
             reference: "/users/" + sessionStorage.getItem("userId") + "/trips/" + sessionStorage.getItem("tripId"),
             items: [],
             tripSetting: {
+                title: "",
                 startDate: new Date(),
                 endDate: new Date(),
-                destinations: []
+                destinations: "",
```

Comment:
why change destinations to a string? it seems more like an array of strings to me

Reply:
it should be an array, but since in the scope of this PR, to sync with the current test data of firebase database and the current code, I changed it to a string just to test things out. However, Emmie's part will ultimately remove this and replace this with her autocomplete which will handle the destinations. So yes, this change will be removed when we come to merge the code.